test/setup.js

```diff
@@ -0,0 +1,30 @@
+import GraphQLJSClient from '../src/graphql-client';
+
+function recordTypes() {
+  const types = GraphQLJSClient.trackedTypes();
+
+  if (typeof require === 'function') {
+    const body = JSON.stringify({'profiled-types': types}, null, '  ');
```

Comment:
I thought it had to be the character literal. I'll use a number, since that's more readable than arbitrary spaces.